README.md

```diff
@@ -72,42 +76,39 @@ The most popular features of Handsontable:
 
 <div id="installation"></div>
 
-## Get started
+## Installation
 
-### 1. Install Handsontable
+#### NPM
 
-#### Using a package manager
-
-Get Handsontable from [npm](https://www.npmjs.com/package/handsontable), [Yarn](https://yarnpkg.com/package/handsontable) or [NuGet](https://www.nuget.org/packages/Handsontable).
+Get Handsontable from your package manager of choice
```

Comment:
why are we removing those links?

Reply:
I definitely want to remove NuGet, and was thinking we shouldn't really be informing users what a package manager is.